src/main/java/com/ericsson/ei/subscription/SubscriptionValidator.java

```diff
@@ -70,18 +75,22 @@ public static void validateSubscription(Subscription subscription) throws Subscr
     }
 
     /**
-     * Validation of subscriptionName parameter Throws SubscriptionValidationException if validation of
-     * the parameter fails due to wrong format of parameter.
+     * Validation of subscriptionName parameter Throws
+     * SubscriptionValidationException if validation of the parameter fails due to
+     * wrong format of parameter.
      *
      * @param subscriptionName
      */
     private static void validateSubscriptionName(String subscriptionName) throws SubscriptionValidationException {
-        // When this regExp need to be changed then remember to change the one in the
-        // back-end (invalidSubscriptionNameRegex in subscription.js), which do the same
-        // invalid subscription name check. The two
-        // regEx always need to be the same for ensuring the same check.
-        // /(\W)/ Is a regEx that matches anything that is not [A-Z,a-z,0-8] and _.
-        String invalidSubscriptionNameRegex = "(\\W)";
+        // Here the regEx used is :/(\W)/ that matches anything that is not
+        // [A-Z,a-z,0-8] and _.
+        String invalidSubscriptionNameRegex = null;
+        try {
+            invalidSubscriptionNameRegex = (String) RegExProvider.getRegExs().get("invalidName");
```

Comment:
You may get it as string directly and there is no need to cats it. 
RegExProvider.getRegExs().getString("invalidName");
Should do the work for you.